app/views/layouts/application.html.erb

```diff
@@ -58,9 +58,26 @@
 <% end %>
 
 <% content_for :footer_support_links do %>
-  <li>
-    <%= link_to(t('.staff_guide'), staff_path) %>
-  </li>
+  <ul>
+    <li>
+      <%= link_to(t('.staff_guide'), '/staff/staffuiguide.md') %>
+    </li>
+    <li>
+      <%= link_to(t('.recent_changes'), '/staff/changes.md') %>
+    </li>
+    <li>
+      <%= link_to(t('.stats'), 'https://staff.prisonvisits.service.gov.uk/en/metrics') %>
```

Comment:
This wont work as expected in staging as it will linking to prod. Instead we can use a url helper. I think `metrics_path` should fix it